code/game/machinery/requests_console.dm

```diff
@@ -464,7 +464,7 @@ GLOBAL_LIST_EMPTY(allConsoles)
 	emergency = null
 	update_icon()
 
-/obj/machinery/requests_console/proc/createmessage(source, title, message, priority)
+/obj/machinery/requests_console/createmessage(source, title, message, priority)
```

Comment:
Why this change in requests_console.dm?

Reply:
linters demanded the parent object have the proc. we don't use request consoles, but this also doesn't hinder their functionality in any way :)

just moves the proc up one level because linters:tm: